.travis.yml

```diff
@@ -3,9 +3,23 @@ sudo: false
 language: go
 
 go:
-- "1.10"
-
+  - "1.10"
+services:
+  - docker
+before_script:
+  - curl -Lo kubectl \https://storage.googleapis.com/kubernetes-release/release/v1.13.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - wget https://github.com/kubernetes-sigs/kind/releases/download/0.2.1/kind-linux-amd64
+  - chmod +x kind-linux-amd64
+  - mv kind-linux-amd64 kind
+  - export PATH=$PATH:$PWD
+  - kind create cluster --config=$TRAVIS_BUILD_DIR/hack/kind_config.yaml
+  - export KUBECONFIG="$(kind get kubeconfig-path)"
+  - docker pull kubernetes/pause
+  - kind load docker-image kubernetes/pause
+  - cp "$(kind get kubeconfig-path --name="kind")" /tmp/admin.conf
 script:
-  - make lint
-  - make build
-  - make test-unit
+- hack/verify-gofmt.sh
+- make lint
```

Comment:
Why this commit coming in? Can you check this @Bowenislandsong

Reply:
this is to accommodate https://github.com/kubernetes-incubator/descheduler/commit/dee89a6cc1ec00ed699a09ed632ca3ed68232a92 It asked for 'gofmt' which I didn't do. I will use 'gofmt' for all the code in the update and then it should not be a problem anymore.